ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/PullQueryExecutor.java

```diff
@@ -141,14 +141,12 @@ public static TableRowsEntity execute(
     }
 
     if (!statement.getConfig().getBoolean(KsqlConfig.KSQL_QUERY_PULL_ENABLE_CONFIG)) {
-      throw new KsqlRestException(
-          Errors.badStatement(
-              "Pull queries are disabled. "
-                  + PullQueryValidator.NEW_QUERY_SYNTAX_SHORT_HELP
-                  + System.lineSeparator()
-                  + "Please set " + KsqlConfig.KSQL_QUERY_PULL_ENABLE_CONFIG + "=true to enable "
-                  + "this feature.",
-              statement.getStatementText()));
+      throw new KsqlException(
```

Comment:
nit: may as well use `KsqlStatementException` and include the statement text

Reply:
We'll keep it as KsqlException for now since changing it would require adding a dep for the test.